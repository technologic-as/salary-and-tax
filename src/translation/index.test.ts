import { getLocales, getMessages } from './index';

describe('translation', () => {
  describe('getMessages', () => {
    it('should return norwegian messages', () => {
      expect(getMessages()).toMatchSnapshot();
    });
  });
  describe('getLocales', () => {
    it('should return all locales', () => {
      expect(getLocales()).toMatchSnapshot();
    });
  });
});
