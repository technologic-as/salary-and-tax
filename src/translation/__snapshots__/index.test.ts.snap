// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`translation getLocales should return all locales 1`] = `
Array [
  Object {
    "active": true,
    "label": "Norsk",
    "value": "nb",
  },
  Object {
    "active": false,
    "label": "English",
    "value": "en",
  },
]
`;

exports[`translation getMessages should return norwegian messages 1`] = `
Object {
  "salary.calculations.employer.fee": "Arbeidsgiveravgift",
  "salary.calculations.header": "Inntekstberegninger",
  "salary.calculations.income": "Brutto inntekt",
  "salary.calculations.pension": "Pensjon",
  "salary.calculations.subcontractor.cut": "Andel til overleverandør",
  "salary.calculations.surplus": "Overskudd",
  "salary.calculations.turnover": "Omsetning",
  "salary.calculations.vacation.savings": "Feriepenger",
  "salary.parameters.employer.fee": "Arbeidsgiveravgift",
  "salary.parameters.header": "Parametere",
  "salary.parameters.hour.rate": "Timepris",
  "salary.parameters.hours.per.year": "Timer per år",
  "salary.parameters.include.in.calculation": "Inkluder i utregningen",
  "salary.parameters.locale": "Språk",
  "salary.parameters.pension": "Pensjon",
  "salary.parameters.pension.step1": "(%) (1G-7.1G)",
  "salary.parameters.pension.step2": "(%) (7.1G-12G)",
  "salary.parameters.percentage": "(%)",
  "salary.parameters.subcontractor": "Overleverandør",
  "salary.parameters.subcontractor.cut": "Andel (%)",
  "salary.parameters.submit": "Beregn",
  "salary.parameters.surplus": "Overskudd",
  "salary.parameters.surplus.amount": "Beløp",
  "salary.parameters.turnover": "Omsetning",
  "salary.parameters.vacation.savings": "Feriepenger",
  "summary.dividends": "Utbytte",
  "summary.header": "Sammendrag",
  "summary.income": "Netto inntekt første år",
  "summary.total.cuts": "Andel til overleverandør, overskudd, arbeidsgiveravgift, feriepenger, pensjon",
  "summary.total.income": "Total inntjening første år (uten feriepenger)",
  "summary.total.tax": "Skatt sammenlagt",
  "summary.turnover": "Omsetning",
  "surplus.graph.chart.header": "Inntekt vs. overskudd",
  "surplus.graph.header": "Inntekt vs. overskudd",
  "surplus.graph.x.axis.title": "Overskudd",
  "surplus.graph.y.axis.title": "NOK",
  "surplus.series.dividends": "Utbytte",
  "surplus.series.gross.seven.g": "7.1G brutto inntekt",
  "surplus.series.income.gross": "Brutto inntekt",
  "surplus.series.income.net": "Netto inntekt",
  "surplus.series.max.total": "Maksimal inntjening",
  "surplus.series.surplus": "Overskudd",
  "surplus.series.total": "Sammenlagt",
  "surplus.suffix": "NOK",
  "tax.calculations.after.tax": "Netto inntekt",
  "tax.calculations.common.income": "Inntektsskatt-grunnlag",
  "tax.calculations.common.income.header": "Inntektsskatt-grunnlag",
  "tax.calculations.dividends.amount": "Utbytte",
  "tax.calculations.dividends.tax": "Skatt på utbytte",
  "tax.calculations.header": "Skatteberegninger",
  "tax.calculations.income": "Brutto inntekt",
  "tax.calculations.income.header": "Inntekt",
  "tax.calculations.income.tax": "Inntektsskatt (22%)",
  "tax.calculations.minimum.deduction": "Minstefradrag (45%)",
  "tax.calculations.personal.allowance": "Personfradrag",
  "tax.calculations.social.security.deduction": "Trygdeavgift (8.2%)",
  "tax.calculations.step.1": "Trinnskatt trinn 1 (1.9%)",
  "tax.calculations.step.2": "Trinnskatt trinn 2 (4.2%)",
  "tax.calculations.step.3": "Trinnskatt trinn 3 (13.2%)",
  "tax.calculations.step.4": "Trinnskatt trinn 4 (16.2%)",
  "tax.calculations.surplus.amount": "Selskapets overskudd",
  "tax.calculations.surplus.header": "Overskudd",
  "tax.calculations.surplus.tax": "Skatt på overskudd (22%)",
}
`;
